.github/workflows/gradle.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.TOKEN_GIT }}
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+    - name: Build with Gradle
+      uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
+      with:
+        arguments: build
+    - name: Run everything
+      run: |
+        sh run-everything
```

Comment:
There is no value value in running the entire benchmark sequence on every PR. 

 1. The entire benchmark takes 5-10 hours to complete iirc, so that's too long. 
 2. We don't know what hardware this runs on, whether we could get CPU throttled for CPU intensive workloads, etc.
 3. Benchmark results don't change on every PR in a meaningful way. Running these benchmarks on at semi-regular intervals is good enough.
 4. The output pipeline -> graph is not automated, so publishing results would still require manual steps.

Please remove this github workflow.

Reply:
1. lowered it to 1.5 hour, with lowering warmupIterations and measurementIterations in Cli.java
2. it is just a give idea and keep track of the history, especially after dependency version changes.

no need to merge this pr without the workflow, otherwise anyone can download the code and run in their own.